dispatch/static/manager/src/js/api/dispatch.js

```diff
@@ -30,8 +30,11 @@ function buildRoute(route, id) {
   return fullRoute
 }
 
-function buildHeaders(token) {
-  let headers = DEFAULT_HEADERS
+function buildHeaders(token, useDefaultHeaders=true) {
+  let headers = {}
+  if(useDefaultHeaders) {
+    headers = DEFAULT_HEADERS
+  }
```

Comment:
This works for now, but after you merge I'm going to modify this method to work better with a change we made in Jimmy's logout branch.